samples/MvvmSampleXF/MvvmSampleXF/Services/SettingsService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using MvvmSample.Core.Services;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Essentials;
+
+namespace MvvmSampleXF.Services
+{
+	public sealed class SettingsService : ISettingsService
+	{
```

Comment:
Nit: same as above, please make sure indentations in all files uses 4 spaces and not tabs 🙂